android/MobiledgeXSDKDemo/README.md

```diff
@@ -16,80 +16,81 @@
 
 #### Perform All
 Select the red button in the lower right to perform all of these API calls in succession.
 
 ### GPS Spoofing Demo
-The app uses the MobiledgeX [VerifyLocation](http://swagger.mobiledgex.net/client/#operation/VerifyLocation "VerifyLocation") API call to verify that the GPS coordinates reported are where the device actually is. 
+The app uses the MobiledgeX [VerifyLocationAPI](http://swagger.mobiledgex.net/client/#operation/VerifyLocation "VerifyLocation") to verify that the GPS coordinates reported are where the device actually is. 
 
 When you start the app, you will see your location on the map represented by a mobile phone icon. 
 
-In the real world, the location verification is implemented by the cellular network operator. For this demo, we use a simulator that stores location data, and is keyed on your current public IP address. To get an initial entry in the simulator database, tap the 3-dot menu, and select "Reset Location". Now the simulator considers your true location to be wherever your current GPS coordinates are. If your IP address stays the same, you won't need to repeat this step next time you run the app.
+In the real world, location verification is implemented by the cellular network operator. For this demo, we use a simulator that stores location data and is keyed on your current public IP address. To get an initial entry in the simulator database, tap the 3-dot menu, and select "Reset Location." Now the simulator considers the valid location to be wherever your current GPS coordinates are. If your IP address stays the same, you won't need to repeat this step next time you run the app.
 
 You can move the mobile phone icon by long-pressing on it and dragging it to a new location, or long-pressing anywhere on the map. When you release the icon, you will be presented with two options:
 - Spoof GPS at this location
 - Update location in GPS database.
 
-If you select "Spoof", this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close this location is to the location stored in the simulator, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
+If you select "Spoof," this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close the location stored in the simulator is, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
```

Comment:
I this really the grammar rule? (School was a long time ago for me :).) I'm using the quotes to represent exactly what is shown on the screen, and there is no comma shown.

Reply:
Quotation Marks:  We use quotation marks with direct quotes, with titles of certain works, to imply alternate meanings, and to write words as words.
Block quotations are not set off with quotation marks.
The quoted text is capitalized if you’re quoting a complete sentence and not capitalized if you’re quoting a fragment.
_Do commas and periods go inside or outside quotation marks?_
Commas and periods always go inside the quotation marks in American English; dashes, colons, and semicolons almost always go outside the quotation marks; question marks and exclamation marks sometimes go inside, sometimes stay outside.

Reply:
Technically, if you use a quote of any kind the writer is  encouraged to cite their source to allow the reader to determine if the quote has been captured correctly.  But, in your favor, I do have a bunch of things happening and I missed a few corrections. Sorry :)

Reply:
*cough* I'm mildly conflicted. It's correct English grammar, but will fail to compile in a coder's mind given it's a guide for new developers. Anyway, carry on everyone :)